tutorials/onchain-governance/fast-track-gov-proposal.md

```diff
@@ -0,0 +1,222 @@
+4. Place decision deposit. This deposit is required to move the referendum from the preparing phase to the deciding phase
+
+```typescript hl_lines="5-14"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+!!!note
+    The [`setCodeWithoutChecks`](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/struct.Pallet.html#method.set_code_without_checks){target=\_blank} extrinsic used in this example is for demonstration purposes only. Replace it with the specific extrinsic that matches your governance proposal's intended functionality. Ensure the call matches the runtime requirements and governance process of your target Polkadot SDK-based network.
+
+???+ function "**generateProposal** (api, call, origin)"
+
+	The `generateProposal` function accomplishes these tasks using a batched transaction, which combines multiple operations into a single transaction:
+
+    1. `preimage.notePreimage` - this submits the preimage of the proposal
+    2. `referenda.submit` - submits the actual proposal to the referenda system
+    3. `referenda.placeDecisionDeposit` - places the required decision deposit for the referendum
+   
+	```typescript
+	--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:generateProposal'
+	```
+
+### Force Proposal Execution
+
+After submitting your proposal, you may want to test its execution without waiting for the standard voting and enactment periods. Chopsticks allows you to force the execution of a proposal by manipulating the chain state and the scheduler.
+
+```typescript hl_lines="16-17"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+???+ function "**forceProposalExecution** (api, call, origin)"
+
+    The `forceProposalExecution` function does the following:
+
+    1. It overwrites the chain storage, modifying the parameters of the proposal to set the approvals and support to the required values for the proposal to pass
+    2. It then forces the scheduler to execute the actual call in the next block, instead of waiting for the original scheduled execution time
+
+    ```typescript
+    --8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:forceProposalExecution'
+    ```
+
+    !!! note
+        This function depends on the `moveScheduledCallTo` utility function, which is crucial for manipulating the scheduler in a forked chain by moving a specific scheduled call to a desired block.
+
+    ??? function "moveScheduledCallTo (api, blockCounts, verifier)"
+
+        ```typescript
+        --8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:moveScheduledCallTo'
+        ```
+
+## Executing the Proposal Script
+
+To run the proposal execution script, use the following command in your terminal:
+
+```bash
+npx ts-node test-proposal.ts
+```
+
+When executing the script, you should expect the following key actions and outputs:
```

Comment:
what about using a terminal element to show the output? or is there nothing really exciting?

Reply:
The script does not output anything, it just executes that logic on chain